docs/cinder-csi-plugin/multi-region-clouds.md

```diff
@@ -101,6 +101,8 @@ parameters:
   csi.storage.k8s.io/node-stage-secret-namespace: kube-system
   csi.storage.k8s.io/provisioner-secret-name: openstack-config-region-one
   csi.storage.k8s.io/provisioner-secret-namespace: kube-system
+  csi.storage.k8s.io/controller-expand-secret-name: openstack-config-region-one
+  csi.storage.k8s.io/controller-expand-secret-namespace: kube-system
```

Comment:
For those, who have questions on how does it work.

These secrets (basically they are not secrets, but names of the cloud region from the cloud-config to use) are read by one of the CSI sidecar controllers (e.g. provisioner, resizer, etc), then passed through the CSI gRPC interface to the Cinder CSI controller / node-server method. CSI controller/node-server has a parsed cloud-config with multiple region names. If the passed cloud region name exists in the config map, it uses it.

In future we'll probably move secrets content into the actual `secret-name` to properly follow the CSI spec way.